spec/environment/environment.rb

```diff
@@ -19,7 +21,17 @@ def load(env = ENV['GD_ENV'] || 'testing')
         puts "USING ENVIRONMENT: #{env}"
         begin
           require_relative env
+          secrets_path = File.join(File.dirname(__FILE__), 'secrets.yaml')
+          secrets = YAML.load_file(secrets_path)
+          env_secrets = secrets[env.downcase].symbolize_keys
+          encryption_key = ENV['GD_SPEC_PASSWORD'] || ENV['BIA_ENCRYPTION_KEY']
+          env_secrets.each do |_, value|
+            decrypted = GoodData::Helpers.decrypt(value, encryption_key)
+            value.replace(decrypted)
+          end
+          GoodData::Environment::ConnectionHelper.set_const('SECRETS', env_secrets)
         rescue
+          require 'pry';binding.pry
```

Comment:
warning: Style/Semicolon: Do not use semicolons to terminate expressions. (rubocop)